curriculum/structure/blocks/lab-game-character-stats.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Build a Game Character Stats Tracker",
+  "isUpcomingChange": true,
+  "dashedName": "lab-game-character-stats",
+  "helpCategory": "Python",
+  "blockLayout": "link",
+  "challengeOrder": [{ "id": "68cabc534d2e741f33a3fe1d", "title": "Step 1" }],
+  "blockType": "lab",
```

`blockType` has been changed to `blockLabel`. You'll need to pull the latest changes from main and make this update.
```suggestion
  "blockLabel": "lab",
```